docs/CONTRIBUTING.md

```diff
@@ -21,34 +18,21 @@ These contributions come in two forms: issues and pull requests.
 
 ## Issues
 
-If you spot anything
-that conflicts with our values,
-opening a GitHub Issue is a great way
-to give us specific feedback.
+If you spot anything that conflicts with our values, opening a GitHub Issue is a great way to give us specific feedback.
 
-To make an issue,
-refer to the [GitHub Issues Workflow](https://github.com/circleci/circleci-docs/wiki/GitHub-Issues-Workflow) wiki page.
+To make an issue, refer to the [GitHub Issues Workflow](https://github.com/circleci/circleci-docs/wiki/GitHub-Issues-Workflow) wiki page.
 
 ## Pull Requests
 
-If you feel motivated,
-you can make documentation changes
-and submit a pull request.
+If you feel motivated, you can make documentation changes and submit a pull request.
 
-For minor changes like typos,
-click "Suggest an edit to this page",
-located at the bottom of each document.
-This will take you to the source file on GitHub,
-where you can submit a pull request for your changes.
+For minor changes like typos, click "Suggest an edit to this page", located at the bottom of each document.
+This will take you to the source file on GitHub, where you can submit a pull request for your changes.
 
-For larger edits or new documents,
-[set up a local environment](local-development.md).
-When you are satisfied with your changes,
-create a pull request from your branch
-by following [GitHub's guide](https://help.github.com/articles/creating-a-pull-request-from-a-fork/).
+For larger edits or new documents, [set up a local environment](local-development.md).
```

Comment:
I thought the local environment no longer existed? I would clarify that someone can fork the repo to work locally, but there is really no "environment" to set up anymore as someone can't run the site locally.

Reply:
Ah it's not in the final diff